iosApp/Survey/Sources/Presentation/Modules/SurveySelection/SurveySelectionContainerView.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  SurveySelectionContainerView.swift
+//  Survey
+//
+//  Created by Bliss on 26/10/22.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import SwiftUI
+
+struct SurveySelectionContainerView: View {
+
+    @State var showingAccountView = false
+
+    var body: some View {
+        ZStack {
+            SurveySelectionView($showingAccountView)
+
+            if showingAccountView {
+                Button {
+                    withAnimation {
+                        showingAccountView = false
+                    }
+                } label: {
+                    ZStack {}
+                        .frame(maxWidth: .infinity, maxHeight: .infinity)
+                }
+            }
+
+            HStack {
+                Spacer()
+                if showingAccountView {
```

Comment:
@blyscuit Should we put this condition wrap the `HStack`? 🤔

Reply:
Declaring this way allow the slide in/out animation when appearing. Following POW when not wrapped this way. 

https://user-images.githubusercontent.com/6356137/210302635-774508cc-d75a-41a3-a8de-e264f4518de8.mov

Reply:
Update POW to include preferred animation.

Reply:
Look good, thanks @blyscuit